### PR TITLE
Add Omi #529

### PR DESCRIPTION
**Pull request by dntzhang**

### CODE REVIEW

**Comment by dntzhang** [comment]
@googlebot I signed it!

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,3 @@ libraries/*/results
 .vscode
 .env
 docs/libraries
-out
```
Comment (rictic) [inline .gitignore:8]: Please revert this change, we don't want to check the out directory into git

File: README.md [hunk 2]
```diff
@@ -23,23 +23,24 @@ npm run build
...
-- [Preact](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/preact)
-- [React](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/react)
-- [Riot.js](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/riot)
-- [Skate](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/skate)
-- [Stencil](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/stencil)
-- [Surplus](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/surplus)
-- [Svelte](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/svelte)
-- [Vue](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/vue)
+## Current List of [Libraries/Frameworks](https://github.com/omijs/cee/tree/master/libraries)
+- [Angular](https://github.com/omijs/cee/tree/master/libraries/angular)
```
Comment (rictic) [inline README.md:27]: It looks like you've converted all URLs into the webcomponents/custom-elements-everywhere repo to point to your fork? Please revert

File: index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script>
+    location.href = 'https://omijs.github.io/cee/out'
```
Comment (rictic) [inline index.html:7]: Please revert

File: libraries/omi/package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "scripts": {
+    "test": "cross-env LIBRARY_NAME=omi karma start",
+    "build": "npm run test"
+  },
+  "devDependencies": {
```
Comment (rictic) [inline libraries/omi/package.json:6]: Please update these deps to their latest versions

File: out/404.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (rictic) [inline out/404.html:1]: Please don't check in the out directory

**Review by rictic — CHANGES_REQUESTED** [review]
Thank you for the change!

Please rebase to `main` and run `npm ci && npm run generate-goldens` to generate an expectedResults.json file for omi, so that future changes don't accidentally regress on its scores

**Comment by dntzhang** [comment]
@rictic All completed.

File: docs/index.handlebars [hunk 6]
```diff
@@ -95,7 +95,7 @@ limitations under the License.
               to how things should work, but they're by no means final. This
               project is also about driving discussion and finding consensus, so
               don't be afraid to
-              <a href="https://github.com/webcomponents/custom-elements-everywhere">open a GitHub issue</a>
+              <a href="https://github.com/omijs/cee">open a GitHub issue</a>
```
Comment (rictic) [inline docs/index.handlebars:98]: revert please

File: docs/index.handlebars [hunk 7]
```diff
@@ -338,7 +338,7 @@ limitations under the License.
               </p>
               <p>
                 If you'd like to contribute, please first
-                <a href="https://github.com/webcomponents/custom-elements-everywhere/issues">open an issue</a>
+                <a href="https://github.com/omijs/cee/issues">open an issue</a>
```
Comment (rictic) [inline docs/index.handlebars:341]: revert please

File: docs/index.handlebars [hunk 8]
```diff
@@ -359,7 +359,7 @@ limitations under the License.
             2.0</a></strong>.
       </p>
       <div class="has-text-centered fork-me-footer">
-        <a href="https://github.com/webcomponents/custom-elements-everywhere">
+        <a href="https://github.com/omijs/cee">
```
Comment (rictic) [inline docs/index.handlebars:362]: revert please

File: docs/partials/octocat.handlebars [hunk 9]
```diff
@@ -8,7 +8,7 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 or implied. See the License for the specific language governing permissions and
 limitations under the License.
 --}}
-<a href="https://github.com/webcomponents/custom-elements-everywhere" class="github-corner" aria-label="View source on
+<a href="https://github.com/omijs/cee" class="github-corner" aria-label="View source on
```
Comment (rictic) [inline docs/partials/octocat.handlebars:11]: revert please

**Review by rictic — CHANGES_REQUESTED** [review]
I expect to see a file at libraries/omi/meta/expectedResults.json

`npm run update-goldens` should generate it

**Comment by dntzhang** [comment]
> expect to see a file at libraries/omi/meta/expectedResults.json
> 
> `npm run update-goldens` should generate it

done

File: libraries/omi/meta/summary.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+<h4 id="omi-handling-data">Handling data</h4>
+
+Omi passes all data to Custom Elements as attributes.
```
Comment (rictic) [inline libraries/omi/meta/summary.md:3]: This doesn't seem right, otherwise the advanced property tests won't pass, will they?

Specifically the test: advanced support -> attributes and properties -> will pass object data as a property

It looks like Omi behaves similarly to Preact, in that it will set the property if it exists on the element, and otherwise it will set an attribute.

File: libraries/omi/src/basic-tests.js [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+
+    it("will pass string data as either an attribute or a property", function () {
+      this.weight = 3;
+      let root = render(<component-with-properties />, scratch);
+      let wc = root.shadowRoot.querySelector("#wc");
+      let data = wc.str || wc.getAttribute("str");
+      expect(data).to.eql("Omi");
+    });
+
+    it('will set boolean attributes on a Custom Element that has not already been defined and upgraded', function () {
```
Comment (rictic) [inline libraries/omi/src/basic-tests.js:105]: These three tests aren't being tested in the other libraries yet, and we want to run the same tests on all libraries. Please remove these three <component-with-unregistered> tests for now and update expectedResults.json (as afterwards the total will be 30 tests instead of 36)

**Review by rictic — CHANGES_REQUESTED** [review]
Looks good! Two final changes and then this can merge

File: libraries/omi/src/basic-tests.js (hunk 11 above)
Comment (dntzhang) [reply libraries/omi/src/basic-tests.js]: done

File: libraries/omi/meta/summary.md (hunk 10 above)
Comment (dntzhang) [reply libraries/omi/meta/summary.md]: done

**Comment by dntzhang** [comment]
@rictic All completed.

**Review by rictic — APPROVED** [review]
Thank you for the contribution!

This will automatically deploy to github pages, and we'll swap over the main domain name from firebase to gh pages in early January with all of the recent updates, including Omi
